numpy/lib/tests/test_financial_expired.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+import numpy as np
+
+
+def test_financial_expired():
+    with pytest.raises(AttributeError, match='NEP 32'):
+        np.fv
```

Comment:
Maybe just put this into `numpy/tests/test_public_api.py`? Although, that file is mostly complex tests, so I don't care much.

Reply:
I thought about that, but `test_public_api.py` seems to have a specific purpose in maintaing the "standard" numpy API.  This new test doesn't seem to fit there.  It didn't fit well anywhere, so I just put it into its own file.

Reply:
Fair enough :).